Dispatcher/UI/Onboarding/OnboardingScreenViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  OnboardingScreenViewController.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 01/06/2023.
+//
+
+import UIKit
+
+class OnboardingScreenViewController: UIViewController {
+
+    @IBOutlet weak var dispatcherLabel: UILabel!
+    @IBOutlet weak var boardingProgressView: UIProgressView!
+    @IBOutlet weak var introductionLabel: UILabel!
+    @IBOutlet weak var nextButton: UIButton!
+    @IBOutlet weak var skipButton: UIButton!
+    @IBOutlet weak var backgroundImageView: UIImageView!
+    
+    var currentPage = 1
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        introductionLabel.text = TextCostants.firstOnBaordingText
+        boardingProgressView.setProgress(0.333, animated: true)
+        boardingProgressView.layer.cornerRadius = 7
+    }
+    
+
+    @IBAction func skipButtonPressed(_ sender: UIButton) {
+        currentPage = 3
+        prepareForThirdPage()
+    }
+    
+  
+    @IBAction func nextButtonPressed(_ sender: UIButton) {
+        currentPage += 1
+        prepareForNextOnboardingPage()
+    }
+    
+    private func prepareForNextOnboardingPage() {
+        switch currentPage {
+        case 2:
+            prepareForSecondPage()
+        case 3:
+            prepareForThirdPage()
+        case 4:
+            self.performSegue(withIdentifier: SegueIdentifiers.fromOnboardingToAuth, sender: self)
+        default:
+            break
+        }
+    }
+    
+    private func prepareForSecondPage() {
```

Comment:
The prepareForSecondPage & prepareForThirdPage are the same in their concept (same 3 lines, different values). Think of a way to have 1 function instead (why if in the future we'll have 5 onboarding screens?).